spec/controllers/concerns/billable_event_trackable_spec.rb

```diff
@@ -21,6 +21,14 @@
   let(:ial_context) { IalContext.new(ial: 1, service_provider: current_sp) }
   let(:request_id) { SecureRandom.hex }
   let(:session_started_at) { 5.minutes.ago }
+  let(:active_profile) do
+    create(
+      :profile, :active, :verified, user: current_user,
+                                    initiating_service_provider: current_sp
+    )
+  end
+  # Do session_started_at and profile_verified_at need to be different, is there some relationship
+  # to be account for in tests?
```

Comment:
`profile_verified_at` is when the user finished IDV.
`session_started_at` is for the current start of this particular session, which may have no IDV